subsys/bluetooth/controller/hci_driver.c

```diff
@@ -163,6 +163,60 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_PERIPHERAL) ||
 
 #define SDC_FAL_MEM_SIZE SDC_MEM_FAL(CONFIG_BT_CTLR_FAL_SIZE)
 
+#if defined(CONFIG_BT_CTLR_CONN_ISO)
+#define SDC_MEM_CIG SDC_MEM_PER_CIG(CONFIG_BT_CTLR_CONN_ISO_GROUPS)
+#define SDC_MEM_CIS \
+	SDC_MEM_PER_CIS(CONFIG_BT_CTLR_CONN_ISO_STREAMS) + \
+	SDC_MEM_ISO_RX_SDU_POOL_SIZE(CONFIG_BT_CTLR_CONN_ISO_STREAMS)
+#define SDC_CIS_COUNT CONFIG_BT_CTLR_CONN_ISO_STREAMS
+#else
+#define SDC_MEM_CIG   0
+#define SDC_MEM_CIS   0
+#define SDC_CIS_COUNT 0
+#endif
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
+#define SDC_MEM_BIS_SINK \
+	SDC_MEM_PER_BIG(CONFIG_BT_CTLR_SYNC_ISO_STREAM_COUNT) +					\
```

Comment:
I'm currently trying to merge a fix to change this to CONFIG_BT_CTLR_SCAN_SYNC_ISO_SET

Reply:
and here https://github.com/nrfconnect/sdk-nrf/pull/12508/files#diff-ca9ee596a131877b2cbc629ccf2e965015db73c3d5cc74be46a064c4bfbf81a8R1019